engine/src/flutter/lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -38,6 +38,46 @@ import 'tabs.dart';
 import 'tappable.dart';
 import 'text_field.dart';
 
+bool unorderedListEqual<T>(List<T>? a, List<T>? b) {
```

Comment:
Does it make sense to put this next to its sister method `listEquals` here:
https://github.com/flutter/flutter/blob/7e2b4e8f86d53d877b5fd8300094e6cf0f012bb2/engine/src/flutter/lib/web_ui/lib/src/engine/util.dart#L539-L552

Reply:
done